<ui version="4.0" >
 <class>LoadFiles</class>
 <widget class="QWidget" name="LoadFiles" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetFileSets" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>631</width>
     <height>131</height>
    </rect>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
   <column>
    <property name="text" >
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Resolution</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>File Format</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Use</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Cutted Set</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Projection</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Classification</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="listWidgetFiles" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>10</y>
     <width>631</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>File Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxFileType" >
      <item>
       <property name="text" >
        <string>DEM</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Clutter</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Other</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>File Format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxFileFormat" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <item>
       <property name="text" >
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>GDALFILE</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>BINFILE</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>ORTFILE</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Projection:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxProjection" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <item>
       <property name="text" >
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>DEG</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>WGS84GC</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>UTM</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Files to be imported:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonImport" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>330</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Import</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>180</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Directory:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="labelDirectory" >
      <property name="text" >
       <string/>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2" >
     <widget class="QPushButton" name="pushButtonBrowse" >
      <property name="text" >
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QWidget" native="1" name="widgetORT" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>0</y>
         <width>381</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title" >
        <string>GroupBox</string>
       </property>
       <widget class="QWidget" name="gridLayout_2" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>20</y>
          <width>361</width>
          <height>62</height>
         </rect>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" colspan="2" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Binary Directory:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>File:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2" >
          <widget class="QLineEdit" name="lineEditORTDirectory" />
         </item>
         <item row="1" column="1" colspan="2" >
          <widget class="QLineEdit" name="lineEditORTFile" >
           <property name="text" >
            <string>FileName</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Central Meridian:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" >
          <widget class="QSpinBox" name="spinBoxORTCentralMeridian" >
           <property name="minimum" >
            <number>-180</number>
           </property>
           <property name="maximum" >
            <number>180</number>
           </property>
           <property name="value" >
            <number>29</number>
           </property>
          </widget>
         </item>
         <item row="0" column="4" >
          <widget class="QPushButton" name="pushButtonORTBrowse" >
           <property name="text" >
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
